scripts/UpdateSubmodules.ps1

```diff
@@ -26,6 +26,7 @@ $ModuleDirectory = Join-Path $RepoRoot "modules"
 Import-Module "$PSScriptRoot/common.psm1" -Scope Local -Force
 
 function Get-GitChanges([string]$Path) {
+    Write-Verbose "Get-GitChanges"
```

Comment:
Is this output intended or left over?

Reply:
left over. The script was hanging for 30 seconds with no output and I was trying to figure out what was causing it